azure-proto-core-test/SkuTests.cs

```diff
@@ -0,0 +1,135 @@
+﻿using azure_proto_core;
+using NUnit.Framework;
+
+namespace azure_proto_core_test
+{
+    class SkuTests
+    {
+        [Test]
+        public void CompareToTests()
```

Comment:
Break these out into tests for each parameter and use parameterized testing like we discussed in our sync today

Reply:
Updated